test/dapp-library.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { Page } from 'puppeteer'
+import { login } from './test-utils/account'
+import { click, openPage, wait, waitForElementText } from './test-utils/page'
+
+const HOST = process.env.CI === 'true' ? 'http://172.18.0.1:1633' : 'http://127.0.0.1:1633'
+const FDP_STORAGE_PAGE_URL = `${HOST}/bzz/${process.env.BLOSSOM_FDP_STORAGE_PAGE_REFERENCE}/`
+
+describe('Dapp interaction with Blossom, using the library', () => {
+  let page: Page
+  const username = 'testuser'
+  const password = 'pass12345'
+
+  describe('fdp-storage tests', () => {
+    beforeAll(async () => {
+      await login(username, password)
+      page = await openPage(FDP_STORAGE_PAGE_URL)
+    })
```

Comment:
I would create the user in the beforeAll hook instead and you don't need sequencer anymore, do you?